pkg/controller/statefulset/stateful_set_control.go

```diff
@@ -23,7 +23,7 @@ import (
 	"k8s.io/klog"
 
 	apps "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
This isn't necessary.

Reply:
Oops - sorry - I've got VSCode set to auto-vet - I'll remove it